.eslintrc.js

```diff
@@ -3,7 +3,25 @@ module.exports = {
     require.resolve(`@yarnpkg/eslint-config`),
     require.resolve(`@yarnpkg/eslint-config/react`),
   ],
-  ignorePatterns: [
-    `packages/plugin-compat/extra/fsevents/fsevents-*.js`,
+  overrides: [
+    {
+      files: [`!packages/*/sources/{index,Plugin}.ts`],
```

Comment:
While the correct value would be `['!packages/*/sources/index.ts', '!packages/yarnpkg-core/sources/Plugin.ts']`, ESLint seems to still match the files if I put both negated patterns, even though [the docs](https://eslint.org/docs/user-guide/configuring/configuration-files#how-do-overrides-work) state that:
> Override blocks can also specify patterns to exclude from matches. If a file matches any of the excluded patterns, the configuration won't apply.